_data/students.yml

```diff
@@ -186,3 +186,10 @@
   twitter: NgDipXunQuang1
   bio: 17, student, competitive programmer, interested in games
   facebook: xuanquang1999
+
+- name: Isabella Berry
+  github: gillarious
+  image: isabellaberry.png
+  twitter: gillarious
+  bio: 16, high school senior, coder and game designer
+  facebook: gillarious
```

Comment:
Add a new line here to fixup the build errors.
You can see your build logs here https://travis-ci.org/fossasia/gci16.fossasia.org/builds/181270456